### DISCUSSION TITLE
Cloud version e-mail templates are sent in the self-hosted version

### DISCUSSION THREAD

**Discussion by zdeneksvarc** [plausible/analytics#2213] [Anything Else]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Self-hosting

### Describe the bug

Email templates in self-hosted version need to be free of commerce cloud related informations. See this one for example:

```
Hey admin, 

You signed up for a free 30-day trial of Plausible, a simple and privacy-friendly website analytics tool. 

To finish your setup for <domain>, you need to install this lightweight line of JavaScript code into your site to start collecting visitor statistics. 

This Plausible script is 45 times smaller than Google Analytics script so you’ll have a fast loading site while getting all the important traffic insights on one single page. 

Do reply back to this email if you have any questions or need some guidance. 

Thanks,
Uku and Marko
--
https://plausible.mona.services
{{{ pm:unsubscribe }}}
```

### Expected behavior

Fixed email templates for self-hosted version.

### Environment

Plausible v1.4.4

**Comment by metmarkosaric** [comment]
thanks for reporting @zdeneksvarc! this works as intended so it's more like a feature request rather than a bug. we keep the issues tab to actual bugs in the product only. i'll convert this to a feature request thread instead

**Reply by KaKi87** [reply]
Hello,
Any news on this ?
Thanks

**Comment by haydenseitz** [comment]
Is there a way to disable these email for self-hosted?

**Comment by maks-io** [comment]
Guys, can you please reconsider this topic? I was very puzzled when i received those emails by my self hosted instance - i don't quite agree with labelling this a "feature request". I would love to create a PR for this but i am not sure that's too clever since i have never written anything with Elixir.

**Reply by ruslandoga** [reply]
👋 @maks-io 

Just a quick check, what version / image tag are you using?

I think this particular email template has already been updated: https://github.com/plausible/analytics/pull/4496

**Reply by ruslandoga** [reply]
> Just a quick check, what version / image tag are you using?

Note that the "latest" version / image tag is v2.1.3: https://github.com/plausible/analytics/discussions/4629

**Reply by maks-io** [reply]
Hi @ruslandoga ,
thanks for the incredibly fast reply!
I have installed the CE as described here: https://github.com/plausible/community-edition/?tab=readme-ov-file#quick-start
It uses version `2.1.3` indeed, i also just checked my image on my server to be sure.

**Reply by ruslandoga** [reply]
Would you be able to post the email contents here? It might be a different template.

**Reply by maks-io** [reply]
Of course, there you go:

![Screenshot 2024-10-01 at 22 15 26](https://github.com/user-attachments/assets/a54e0d06-509d-4e6d-8232-689d99d9aced)

**Reply by ruslandoga** [reply]
Thank you! I was wrong, it's a [different template](https://github.com/plausible/analytics/blob/v2.1.3/lib/plausible_web/templates/email/site_setup_help_email.html.eex) from the one modified in the PR I linked above. I'll update it in https://github.com/plausible/analytics/pull/4644.
